### PR TITLE
Create why-code-reviews-are-hard.md

### PR DESCRIPTION
**Pull request by content-hacker**
add text

### CODE REVIEW

**Comment by content-hacker** [comment]
@rebeccadee  this is ready for you

File: blogposts/2021/why-code-reviews-are-hard.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+---
+title: Why are code reviews still so hard?
+description: Code reviews are part of the job, so why are they still so tedious? Here's how to make the process better. 
+author: Florian Rappl
+authorUrl: https://twitter.com/florianrappl?lang=en
+publishDate: 2021-06-28T10:00-07:00
+tags: [blog], [code review]
```
Comment (rebeccadee) [inline blogposts/2021/why-code-reviews-are-hard.md:7]:
```suggestion
tags: [blog, code review]
```

**Comment by content-hacker** [comment]
@rebeccadee can this move to the scheduled column?

**Comment by rebeccadee** [comment]
> @rebeccadee can this move to the scheduled column?

@content-hacker I haven't reviewed yet, just pushed some fixes so the preview would work 😁 Will move it over when I've reviewed, thanks for reminding me!

File: blogposts/2021/why-code-reviews-are-hard.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+A solution like Sourcegraph allows you to enhance existing tools such as GitHub or GitLab with universal search integration. Integrations are a vital part of the expected developer experience. Without the necessary integrations, developers may need to switch tooling.
+
+With universal code search running on GitHub, more context is available directly in the code review. Method or function names link to corresponding sections of the codebase. There’s no need to keep a separate copy of the code open in your editor or integrated development environment to follow along and explore the changes.
+
+Search features in integrated development environments and text editors aren’t always up to the task of finding what’s needed. For example, code might be present many times, in different locations, in a database. Universal code search’s ability to spot similar code or duplicates is useful: it identifies existing code much better and infers potential refactoring paths.
+
+These universal code search abilities also reduce the impact of large code reviews. Reviewers can see and understand larger change sets.
```
Comment (rebeccadee) [inline blogposts/2021/why-code-reviews-are-hard.md:85]: @content-hacker I'm concerned about this section going a bit too hard on pushing Sourcegraph and putting people off. I'd like to cut it down a bit and see if we can introduce 1 or 2 other suggestions for improving context in code reviews. WDYT?

**Review by rebeccadee — COMMENTED** [review]
@content-hacker Made some edits and left one comment for you to check out. I also added in some more subheads to break it up and make skimming easier (we know people will! 😆 )

File: blogposts/2021/why-code-reviews-are-hard.md (hunk 2 above)
Comment (rebeccadee) [reply blogposts/2021/why-code-reviews-are-hard.md:85]: @content-hacker After our 1:1 today I'm thinking we should hold off on this post to see if it aligns with our current thinking/how we might be able to improve on it.

**Pull request closed without merging**